main/src/main/scala/sbt/internal/AbstractTaskProgress.scala

```diff
@@ -0,0 +1,81 @@
+/*
+ * sbt
+ * Copyright 2011 - 2018, Lightbend, Inc.
+ * Copyright 2008 - 2010, Mark Harrah
+ * Licensed under Apache License 2.0 (see LICENSE)
+ */
+
+package sbt
+package internal
+
+import java.util.concurrent.ConcurrentHashMap
+import scala.collection.concurrent.TrieMap
+import scala.collection.JavaConverters._
+
+abstract class AbstractTaskExecuteProgress extends ExecuteProgress[Task] {
```

Comment:
Maybe private[sbt] this even though it's in the internal package?

Reply:
Yea. That probably makes sense.